composer.json

```diff
@@ -55,10 +55,5 @@
             "@composer make-fixture",
             "phpunit ./tests --debug"
         ]
-    },
-    "config": {
-        "platform": {
-            "php": "7.3.0"
```

Comment:
Does this leave us with no php constraint? why not 8.0.0?

Reply:
This was never a constraint, it just helped influence what ended up in our lock file. We could add a constraint to the `require` section, but...why? We're already getting that from PHP-TUF.

IMHO we should support whatever PHP-TUF itself supports, rather than have our own opinion about this. So it makes sense, I think, to inherit the constraint.